test/unit/messaging/send-message.test.js

```diff
@@ -0,0 +1,53 @@
+const mockSendMessage = jest.fn()
+const mockCloseConnection = jest.fn()
+const MockMessageSender = jest.fn().mockImplementation(() => {
+  return {
+    sendMessage: mockSendMessage,
+    closeConnection: mockCloseConnection
+  }
+})
+
+jest.mock('ffc-messaging', () => {
+  return {
+    MessageSender: MockMessageSender
+  }
+})
+
+jest.mock('../../../app/messaging/create-message')
+const { createMessage: mockCreateMessage } = require('../../../app/messaging/create-message')
+
+const message = require('../../mocks/messaging/message')
+const paymentRequest = require('../../mocks/payment-requests/payment-request')
+
+const { PROCESSING } = require('../../../app/constants/messages')
+
+const { messageConfig } = require('../../../app/config')
+
+const { sendMessage } = require('../../../app/messaging/send-message')
```

Comment:
Do we want to keep these separated?

Reply:
I  kept sendMessage separate as it's the focus of the test so wanted it to stand out